openid-sharedsignals-framework-1_0.md

```diff
@@ -2041,200 +2213,6 @@ consented to be released by the user, then the Transmitter MUST obtain consent
 to release such data from the user in accordance with the Transmitter's privacy
 policy.
 
-# Profiles {#profiles}
-This section is a profile of the following IETF Security Event specifications:
-
-* Security Event Token (SET) {{RFC8417}}
-* Push-Based SET Token Delivery Using HTTP {{RFC8935}}
-* Poll-Based SET Token Delivery Using HTTP {{RFC8936}}
-
-The RISC use cases that set the requirements are described in Security Events
-RISC Use Cases {{USECASES}}.
-
-The CAEP use cases that set the requirements are described in CAEP Use Cases (TODO: Add
-        reference when file is added to repository.)
-
-## Security Event Token Profile {#set-profle}
-This section provides SSF profiling specifications for the Security Event Token (SET)
-{{RFC8417}} spec.
-
-### Signature Key Resolution {#signature-key-resolution}
-The signature key can be obtained through "jwks_uri", see {{discovery}}.
-
-### SSF Event Subject {#event-subjects}
-The primary Subject Member of SSF events is described in the "Subject Members" section ({{subject-ids}}). The JWT "sub" claim MUST NOT be present in any SET containing
-an SSF event.
-
-### SSF Event Properties {#event-properties}
-The SSF event MAY contain additional claims within the event payload that are
```

Comment:
this was removed. i do think that is ok since additional claims are allowed in SETs, so not normative

Reply:
The section "Event type specific fields" is supposed to be a rephrasing of this sentence that was deleted.